docs/topics/loaders.rst

```diff
@@ -675,3 +675,18 @@ Here is a list of all built-in processors:
     constructor keyword arguments are used as default context values. See
     :class:`Compose` processor for more info.
 
+.. class:: JmesProcessor(json_path)
+
+    Queries the value using the json path provided to the constructor and returns the output. 
+    Requires jmespath (https://github.com/boto/jmespath) to run.
+
+    This processor takes only one string at a time and will return a python string/dictionary/None as answer
```

Comment:
1. Can it return a list? A number? A boolean?
2. What is a "python string"? Does this processor return bytes or unicode?